### PR TITLE
Feature Additions

### PR DESCRIPTION
**Pull request by jsexauer**
- Add confirmation card when completing a task
- Add counts in subtitle for habits
- Add streaks in subtitle for dailies
- Add due date in subtitle for todos
- Add support for timeline to show dailies and todos

### CODE REVIEW

File: appinfo.json [hunk 1]
```diff
@@ -29,31 +29,35 @@
...
     "sdkVersion": "3",
     "shortName": "Habitica Tasks",
     "targetPlatforms": [
         "aplite",
-        "basalt",
-        "chalk"
+        "basalt"
     ],
-    "uuid": "0eb30e8c-0040-40e5-a72b-47ddf0fba3c9",
+    "uuid": "b4754974-0298-42ad-b3f9-bded899f987f",
```
Comment (jsexauer) [inline appinfo.json:60]: I had to change the uuid for testing.  You will need to change it back.  Also be sure to turn on timeline support in the pebble dev portal for this app.

**Comment by kdemerath** [comment]
It looks like you also added #12 and #14 
I somehow miss to see how many checklist items are checked off but I see the priority in the new information placed into the subtitle.

**Comment by jsexauer** [comment]
Haha, I didn't even know those were pending issues :)

**Comment by kdemerath** [comment]
Merged and got several issues on the aplite platform with the timeline function:
[HANDY] pebble-app.js:?: JS: Habitica Tasks: Timeline Failed: Unable to PUT to timeline Error:"{\"errorCode\":\"INVALID_USER_TOKEN\",\"errorMessage\":\"X-User-Token is missing or invalid\",\"requestId\":\"someID\"}" Status: 410 Request: {"readyState":4,"timeout":0,"DONE":4,"status":410,"HEADERS_RECEIVED":2,"responseType":"","LOADING":3,"response":"{\"errorCode\":\"INVALID_USER_TOKEN\",\"errorMessage\":\"X-User-Token is missing or invalid\",\"requestId\":\"SameID\"}","responseText":"{\"errorCode\":\"INVALID_USER_TOKEN\",\"errorMessage\":\"X-User-Token is missing or invalid\",\"requestId\":\"SameIDAgain\"}","UNSENT":0,"statusText":"410","OPENED":1}

**Comment by jsexauer** [comment]
Did you change the UUID back to your UUID?

Did you enable timeline in the pebble dev portal?  See "Enabling New App" section here: https://developer.pebble.com/guides/pebble-timeline/

**Comment by kdemerath** [comment]
I just deployed for testing purpose via cloudpebble.net. I missed to activate via dev portal. I switched the app id back to the old one.

**Comment by jsexauer** [comment]
Even for testing purposes you need to activate the timeline via the dev
portal

**Comment by kdemerath** [comment]
I did the activation but still receiving the error. I saw there are issues with multiple registered Pebbles on the iOS App. I have to delete the Watch App from Pebbles and maybe delete the iOS App aswell.
I keep on testing during the next days.

**Pull request merged**